Px.Utils/PxFile/Metadata/PxFileMetadataReader.cs

```diff
@@ -138,7 +134,7 @@ public async IAsyncEnumerable<KeyValuePair<string, string>> ReadMetadataAsync(
                             throw new InvalidPxFileMetadataException($"Unexpected character '{parsingBuffer[i]}' found at position {i}.");
                         }
 
-                        if (parsingBuffer[i] == nextDelimeter)
+                        if (parsingBuffer[i] == keywordSeperator || parsingBuffer[i] == sectionSeparator)
```

Comment:
This changes the behaviour of the function. Now both separator types are accepted all the time. I would double check if this is actually related to the original issue.

Reply:
Change reverted